fmt/format.h

```diff
@@ -582,6 +590,26 @@ class BasicStringRef {
   }
 #endif
 
+#if FMT_HAS_EXPERIMENTAL_STRING_VIEW
+  /**
+  \rst
```

Comment:
nit: Please indent the rst block for consistency with the rest of the code.

Reply:
ok